1 file changed, 3 insertions(+), 9 deletions(-)
build/packaging/srpm/condor.spec | 12 +++---------

modified   build/packaging/srpm/condor.spec
@@ -985,8 +985,8 @@ rm -rf %{buildroot}/%{_sysconfdir}/init.d
 
 %if %systemd
 # install tmpfiles.d/condor.conf
-mkdir -p %{buildroot}%{_sysconfdir}/tmpfiles.d
-install -m 0644 %{buildroot}/etc/examples/condor-tmpfiles.conf %{buildroot}%{_sysconfdir}/tmpfiles.d/condor.conf
+mkdir -p %{buildroot}%{_tmpfilesdir}
+install -m 0644 %{buildroot}/etc/examples/condor-tmpfiles.conf %{buildroot}%{_tmpfilesdir}/%{name}.conf
 
 mkdir -p %{buildroot}%{_unitdir}
 install -m 0644 %{buildroot}/etc/examples/condor.service %{buildroot}%{_unitdir}/condor.service
@@ -1165,7 +1165,7 @@ rm -rf %{buildroot}
 %dir %_sysconfdir/condor/
 %config(noreplace) %_sysconfdir/condor/condor_config
 %if %systemd
-%config(noreplace) %_sysconfdir/tmpfiles.d/%{name}.conf
+%{_tmpfilesdir}/%{name}.conf
 %{_unitdir}/condor.service
 # Disabled until HTCondor security fixed.
 # %{_unitdir}/condor.socket
@@ -1411,14 +1411,9 @@ rm -rf %{buildroot}
 %dir %_var/lib/condor/execute/
 %dir %_var/log/condor/
 %dir %_var/lib/condor/spool/
-%if %systemd
-%ghost %dir %_var/lock/condor/
-%ghost %dir %_var/run/condor/
-%else
 %dir %_var/lock/condor
 %dir %_var/lock/condor/local
 %dir %_var/run/condor
-%endif
 
 #################
 %files procd
@@ -1802,7 +1797,6 @@ fi
 if [ $1 -eq 1 ] ; then
     # Initial installation 
     /bin/systemctl daemon-reload >/dev/null 2>&1 || :
-    /bin/systemd-tmpfiles --create /etc/tmpfiles.d/%{name}.conf 2>&1 || :
 fi
 
 %preun

